docs/en/02_Developer_Guides/00_Model/11_Scaffolding.md

```diff
@@ -196,9 +196,9 @@ class MyDataObject extends DataObject
     ];
     
     private static $summary_fields = [
-        'Name' => 'Name',
```

Comment:
The old behaviour wasn't incorrect; You can use field_labels to override summary_fields, but it's not necessary to.

Did you find the previously documented API didn't actually work?